tests/foreman/cli/test_environment.py

```diff
@@ -242,18 +239,11 @@ def test_positive_sc_params_by_id(self):
 
         @Assert: The command runs without raising an error
 
-        @BZ: 1219934
-
         """
         environment = make_environment()
-        Environment.sc_params({
-            'environment-id': environment['id'],
-        })
-        if not bz_bug_is_open(1219934):
-            self.fail('BZ #1219934 is closed, should assert the content')
+        Environment.sc_params({'environment-id': environment['id']})
```

Comment:
Never mind about this comment. The reason is written here https://github.com/SatelliteQE/robottelo/pull/3397#discussion_r60477883